locales/en/kafka.json

```diff
@@ -82,6 +88,11 @@
     "uncosumed_partitions_for_topic": "Unconsumed partitions for this topic",
     "view_partitions_offsets": "View partition offsets"
   },
+  "delete_instance_message": "The <0>{{instanceName}}</0> Kafka instance and all topics will be deleted. Applications will no longer have access to the topics in this instance.",
+  "delete_instance_title": "Delete instance?",
+  "delete_topic_active_message": "One or more consumers are currently active in this topic, and will be removed at the time of deletion. ",
+  "delete_topic_message": "The <0>{{instanceName}}</0> topic will be deleted and removed from this instance. Applications will no longer have access to this topic.",
+  "delete_topic_title": "Delete topic",
```

Comment:
I think this string should have a "?" at the end. There is one missing in the modal title:
![image](https://user-images.githubusercontent.com/21063328/178009491-53b71ce0-2ad4-48b9-ad0d-a6a755d83e17.png)

Reply:
Yeah I will fix that now